{{cookiecutter.project_name}}/notebooks/ModelValidation.py

```diff
@@ -160,7 +160,7 @@ def get_targets_from_recipe():
 
 def log_to_model_description(run, success):
     run_info = run.info
-    run_link = "#mlflow/experiments/{0}/runs/{1}".format(run_info.experiment_id, run_info.run_id)
+    run_link = "[Link](#mlflow/experiments/{0}/runs/{1})".format(run_info.experiment_id, run_info.run_id)
```

Comment:
Instead of `Link` maybe something more descriptive like
```suggestion
    run_link = "[Run](#mlflow/experiments/{0}/runs/{1})".format(run_info.experiment_id, run_info.run_id)
```

Reply:
Also, have we confirmed this link works and takes us to the expected run?

Reply:
Yes it does.